Changes to 3.3

jesseli2002 opened the pull request:
- Reworked 3.3.7 a); it's a bit brute force plug and chug but at least seemed to me to be straightforward. Also added in the induction step.
- Simplified the argument for 3.3.11 (used a simpler example, which I assume counts as the "clean this up" TODO?)

UlisseMini reviewed (commented):
I think it would be good to mention how `A + A = {x+y : x,y ∈ A}` since it seems unclear how set addition is defined. Also the `C_{n+1}'` stuff is confusing, but a confusing proof is better then no proof I guess

jesseli2002 commented:
Strictly, `A + A = {x+y : x,y ∈ A}` is defined in the problem statement itself, but I've added in a definition anyways to make clear the case for `A + B` (when the two sets are different). 

The `C_{n+1}'` comes from noticing that upscaling `C_{n+1}` by a factor of 3 leaves you with two copies of `C_n`; the rest is formalizing the argument. I've added some notes explaining that intuition.

jesseli2002 commented:
> I think it would be good to mention how `A + A = {x+y : x,y ∈ A}` since it seems unclear how set addition is defined. Also the `C_{n+1}'` stuff is confusing, but a confusing proof is better then no proof I guess

Strictly, `A + A = {x+y : x,y ∈ A}` is defined in the problem statement itself, but I've added in a definition anyways to make clear the case for `A + B` (when the two sets are different). 

The `C_{n+1}'` comes from noticing that upscaling `C_{n+1}` by a factor of 3 leaves you with two copies of `C_n`; the rest is formalizing the argument. I've added some notes explaining that intuition.
